hazelcast-spring/src/main/resources/hazelcast-spring-3.8.xsd

```diff
@@ -314,13 +460,14 @@
                                     <xs:element name="partition-strategy" type="xs:string" minOccurs="0" maxOccurs="1"/>
                                 </xs:sequence>
                                 <xs:attribute name="name" use="required" type="xs:string"/>
-                                <xs:attribute name="in-memory-format" use="optional" type="xs:string" default="BINARY">
+                                <xs:attribute name="in-memory-format" use="optional" type="in-memory-format" default="BINARY">
```

Comment:
Difficult since Spring fails when converting a value which is not a valid enum value - `Cannot convert value of type [java.lang.String] to required type [com.hazelcast.config.InMemoryFormat] for property 'inMemoryFormat': no matching editors or conversion strategy found`